src/tplr/comms.py

```diff
@@ -777,7 +785,7 @@ async def get(
         time_max: datetime = None,
     ) -> Optional[tuple[dict, int]]:
         """GET operation."""
-        filename = f"{key}-{window}-{uid}-v{__version__}.pt"
+        filename = f"{key}-{window}-{uid}-v{self.checkpoint_version}.pt"
```

Comment:
I think this is problematic. We try to gather from `self.checkpoint_version`. But miner upload for `__version__` right?